cmd2/cmd2.py

```diff
@@ -3489,11 +3499,27 @@ def do_load(self, args: argparse.Namespace) -> None:
             # command queue. Add an "end of script (eos)" command to cleanup the
             # self._script_dir list when done.
             with open(expanded_path, encoding='utf-8') as target:
-                self.cmdqueue = target.read().splitlines() + ['eos'] + self.cmdqueue
+                script_commands = target.read().splitlines()
         except OSError as ex:  # pragma: no cover
             self.perror("Problem accessing script from '{}': {}".format(expanded_path, ex))
             return
 
+        if args.record_transcript:
+            if self.redirecting:
```

Comment:
This isn't necessary because the output of the load command still will be redirected after the transcript is generated. The `pfeedback()` call in `_generate_transcript()` will write to the redirected location.

Reply:
Ok.  I took the guards out.  I'll probably wait for your change to merge in before I merge this PR.